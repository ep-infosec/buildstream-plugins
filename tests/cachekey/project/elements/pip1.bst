kind: pip
sources:
- kind: local
  path: files/local
