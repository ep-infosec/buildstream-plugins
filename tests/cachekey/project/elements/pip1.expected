321bf2ac1384c306b7be050c163ff21a22ba672c6061786bb4015c62abe2ea3f